api/src/aggregate/aggregate.service.ts

```diff
@@ -73,4 +83,30 @@ export class AggregateService {
       return null
     }
   }
+
+  async findLatestByAggregatorId(latestAggregateByIdDto: LatestAggregateByIdDto) {
+    const { aggregatorId } = latestAggregateByIdDto
+
+    const redisKey = `latestAggregate:${aggregatorId.toString()}`
+    const rawResult = await this.redis.get(redisKey)
+
+    if (!rawResult) {
+      return await this.findLatestByAggregatorIdFromPrisma(aggregatorId)
+    }
+    const { timestamp, value } = JSON.parse(rawResult)
+    return { timestamp, value: BigInt(value) }
```

Comment:
I think we should be returning this in `string` format. When we use `string`, we don't have to serialize it and it can be consumed directly by client without knowing how to handle bigints 🤔 What do you think?

Reply:
I tried to minimize the client code changes. But if you insist, I'll modify it that way.